projects/kit/components/input-time/input-time.template.html

```diff
@@ -49,3 +49,17 @@
         </button>
     </tui-data-list>
 </ng-template>
+
+<ng-template #iconContent>
+    <tui-svg
+        *polymorpheusOutlet="icon as src; context: {$implicit: size}"
+        tuiWrapper
+        appearance="icon"
+        [class.t-icon]="!computedDisabled"
+        [src]="src"
+    ></tui-svg>
+    <input
+        *ngIf="isMobile && nativePicker"
```

Comment:
We can also use `datalist` for items, it works for Android:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist#date_and_time_types